src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -165,6 +165,9 @@ export default class ChargingStationService {
       }
       chargingStation.public = filteredRequest.public;
     }
+    if (Utils.objectHasProperty(filteredRequest, 'tariffID')) {
```

Comment:
This bit was added by @rmangin-fr. 
I am also not sure why we check it like this.

Reply:
Copy/Paste without thinking ;-)
This is used only for boolean prop check.
You can use a simple IF statement.
Thanks!

Reply:
@LucasBrazi06 
I think the has property check is required if we want to be able to clear the tariffID. 
In this case the value of tariffID is null and if we check with IF statement the null value will set the statement false and it is not cleared in the db.

Reply:
@OliveGerste
You're absolutely right!
In the new Auth framework this will be filtered out by the fields you have the right to see then this code will no longer be necessary but ChargingStationService has not yet been migrated :-( !
My bad!